tools/breeze/src/main.rs

```diff
@@ -0,0 +1,78 @@
+use std::{thread, time};
+
+use cgos::board::BoardClass;
+use cgos::congatec::Congatec;
+use cgos::status::Status;
+
+static FAN_MAX_SPEED: f32 = 100.0;
+static FAN_MIN_SPEED: f32 = 30.0;
+static INTERPOLATION_X0: f32 = 50.0;
+static INTERPOLATION_Y0: f32 = 50.0;
+static INTERPOLATION_X1: f32 = 70.0;
+static INTERPOLATION_Y1: f32 = 100.0;
+static SLEEP_DURATION_SEC: u64 = 30;
+
+#[derive(Debug, Clone)]
```

Comment:
Sort alphabetically?

```suggestion
#[derive(Clone, Debug)]
```